consoles/sshVirtsh.pm

```diff
@@ -496,6 +498,22 @@ sub resume ($self) {
 
 sub get_remote_vmm ($self) { $bmwqemu::vars{VMWARE_REMOTE_VMM} // '' }
 
+sub _encode_config ($self, $config, $key) {
+    # expand path
+    $config = sprintf('%s/data/%s', $bmwqemu::vars{CASEDIR}, $config);
```

Comment:
```suggestion
    $config = "$bmwqemu::vars{CASEDIR}/data/$config";
```

Reply:
I honestly prefer `sprintf` formatting than mangling it altogether with concatenation.

Reply:
Really? For me the interpolation is much more readable because one doesn't have to shuffle around the pieces together in one's head. (With `sprintf` the middle part of the code becomes the start of the string.)

Reply:
I admit, it is a matter of taste. I prefer it as you can clearly see what is the string literal and what is going to be replaced and in which order, and is usually more spacious than a dense long concatenation.

Reply:
I also find interpolation easier to read. Maybe even go a step further:

```suggestion
    my $casedir = $bmwqemu::vars{CASEDIR};
    $config = "$casedir/data/$config";
```

Reply:
Alright, added. Let's move on :)